frontend/scripts/componentSchemas/schemaUtils.test.ts

```diff
@@ -0,0 +1,65 @@
+import { ensureTypeWithEnums } from './schemaUtils';
+
+describe('ensureTypeWithEnums', () => {
+  it('should set schema.type to "string" when schema.enum contains a string', () => {
+    const schema: any = {
```

Comment:
Can we use something else than `any` type here as well? 😄

Reply:
Fixed